code/game/objects/items/rogueitems/natural/wood.dm

```diff
@@ -69,7 +69,7 @@
 		lumber_amount = rand(minimum, max(round(skill_level), minimum))
 		var/essense_sound_played = FALSE //This is here so the sound wont play multiple times if the essense itself spawns multiple times
 		for(var/i = 0; i < lumber_amount; i++)
-			if(prob(skill_level+ user.goodluck(2)))
+			if(prob(skill_level + CLAMP((user.STALUC - 10)*2,0,100)))
```

Comment:
same her3e

Reply:
This is actually, accidentally a buff!
User.goodluck 2 would give a 2% chance per luck to return TRUE, so `prob(skill_level + TRUE)` or prob(skill_level + 1)
The new function would give a 2% chance per luck over 10 to give one, so ill fix this anyway